godel_robots/abb/godel_irb2400/godel_irb2400_support/launch/irb2400_blending.launch

```diff
@@ -37,15 +48,33 @@
       <rosparam command="load" file="$(find godel_irb2400_support)/config/point_cloud_descriptions.yaml"/>
     </node>
   </group>
+
+  <!-- If the user uses the real sensor, launch the ensenso publisher node -->
+  <group unless="$(arg sim_sensor)">
+    <remap from="/depth/points" to="/sensor_point_cloud"/>
+    <param name="camera_frame_id" type="string" value="tool0" />
+    <node name="ensenso_publisher_node" pkg="ensenso_publisher" type="ensenso_publisher_node" output="screen" required="true">
```

Comment:
My comment was wrong/unclear, I created an issue to detail what I think is wrong:
https://github.com/Jmeyer1292/ensenso_publisher/issues/1